yt/funcs.py

```diff
@@ -814,7 +814,12 @@ def fix_axis(axis, ds):
 
 def get_image_suffix(name):
     suffix = os.path.splitext(name)[1]
-    return suffix if suffix in ['.png', '.eps', '.ps', '.pdf'] else ''
+    supported_suffixes = ['.png', '.eps', '.ps', '.pdf', '.jpg', '.jpeg']
+    if suffix in supported_suffixes or suffix == '':
+        return suffix
+    else:
+        mylog.warning('Unsupported image suffix requested (%s)' % suffix)
```

Comment:
so if I don't supply a suffix am I going to get this warning every time?

Reply:
if you don't supply a suffix it should be handled in the `if` part of this conditional I think. I just saved without adding a suffix and it chose 'png' for me without displaying this warning.

Reply:
Oh, you're right.  Never mind.

Reply:
That’s right, is.path.splittext will return 2 strings no matter what, but the second is empty when there’s no suffix to be isolated